### PR TITLE
Extract constants in WebUtils

### PR DESCRIPTION
**Pull request by xiaoheng1**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

Extract constants in WebUtils

see more detail #5676 

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

**Comment by KomachiSion** [comment]
CI can't pass

**Comment by xiaoheng1** [comment]
> CI can't pass

done

File: core/src/main/java/com/alibaba/nacos/core/utils/WebUtils.java [hunk 1]
```diff
@@ -43,6 +43,18 @@
  */
 public class WebUtils {
     
+    private static final String ENCODING_KEY = "encoding";
+    
+    private static final String ACCEPT_CHARSET_KEY = "Accept-Charset";
```
Comment (Maijh97) [inline core/src/main/java/com/alibaba/nacos/core/utils/WebUtils.java:48]: Can be replaced with
`com.alibaba.nacos.common.constant.HttpHeaderConsts#ACCEPT_CHARSET`

File: core/src/main/java/com/alibaba/nacos/core/utils/WebUtils.java [hunk 2]
```diff
@@ -43,6 +43,18 @@
...
+    
+    private static final String COMMA = ",";
+    
+    private static final String SEMI = ";";
+    
+    private static final String JSON_AND_UTF8_CONTENT_TYPE = "application/json;charset=UTF-8";
```
Comment (Maijh97) [inline core/src/main/java/com/alibaba/nacos/core/utils/WebUtils.java:54]: Can be replaced with
`com.alibaba.nacos.common.http.param.MediaType#APPLICATION_JSON`

File: core/src/main/java/com/alibaba/nacos/core/utils/WebUtils.java (hunk 1 above)
Comment (xiaoheng1) [reply core/src/main/java/com/alibaba/nacos/core/utils/WebUtils.java:48]: done

File: core/src/main/java/com/alibaba/nacos/core/utils/WebUtils.java (hunk 2 above)
Comment (xiaoheng1) [reply core/src/main/java/com/alibaba/nacos/core/utils/WebUtils.java:54]: done

**Review by KomachiSion — APPROVED** [review]

**Pull request merged**